__tests__/app.test.js

```diff
@@ -41,8 +41,46 @@ describe("GET - /api", () => {
       .get("/api")
       .expect(200)
       .then(({ body }) => {
-        console.log(body);
         expect(body.endpointData).toEqual(endpointData);
       });
   });
 });
+
+describe("GET -  /api/articles/:article_id", () => {
+  test("200: Responds with article object with the specified article id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+
+        expect(article).toHaveLength(1);
```

Comment:
If you only want one article, theres no need to send it back on an array, I'll point out where else to change this further down >>>